subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -314,12 +322,11 @@ void radio_phy_set(uint8_t phy, uint8_t flags)
 				RADIO_MODECNF0_RU_Pos) &
 			       RADIO_MODECNF0_RU_Msk);
 #else /* !CONFIG_BT_CTLR_RADIO_ENABLE_FAST */
-#if !defined(CONFIG_SOC_SERIES_NRF51X)
 	NRF_RADIO->MODECNF0 = (RADIO_MODECNF0_DTX_Center <<
 			       RADIO_MODECNF0_DTX_Pos) &
 			      RADIO_MODECNF0_DTX_Msk;
-#endif /* !CONFIG_SOC_SERIES_NRF51X */
 #endif /* !CONFIG_BT_CTLR_RADIO_ENABLE_FAST */
+#endif /* !CONFIG_SOC_SERIES_NRF51X && !CONFIG_SOC_SERIES_NRF54LX */
 }
 
 void radio_tx_power_set(int8_t power)
```

Comment:
power will need to be int16_t at least (TXPOWER uses 9 bits) (unless you start by limiting this to values >= -26dBm and hope the mapping does not move out of range for other 54L devices)
Same applies to radio_tx_power_max_set/radio_tx_power_min_get / radio_tx_power_floor()
and `lll_radio_tx_pwr_floor()` and lll_test:tx_power_set() and wherever this propagates to..

Reply:
Updated